server/src/db/migrations/20201130105446_create_warm_fuzzies.cjs

```diff
@@ -0,0 +1,35 @@
+const tableName = "warmFuzzies";
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  const tableExists = await knex.schema.hasTable(tableName);
+
+  if (!tableExists) {
+    console.log(`Creating ${tableName}`)
+    return knex.schema.createTable(tableName, (table) => {
+      table.bigIncrements("id");
+      table.string("title").notNullable();
+      table.string("description").notNullable();
```

Comment:
we may want to consider using a `text` column here if we expect there to be a long comment about the fuzzy. we're limited to 255 characters

Reply:
Story created